src/test/java/io/confluent/idesidecar/websocket/resources/FlinkLanguageServiceProxyTest.java

```diff
@@ -131,7 +132,7 @@ void setup() {
         .thenReturn(mockedConnection);
   }
 
-  @Test
+  @RetryingTest(maxAttempts = 3)
```

Comment:
Marked this test as retryable until https://github.com/confluentinc/ide-sidecar/issues/404 has been resolved.